docs/admin/addons.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
```

Comment:
"Pluggable" is a weird word. Industry term of art?

Maybe "Add-ons are plug-in modules that extend Kubernetes' functionality."